src/view/EditProfileView.java

```diff
@@ -0,0 +1,267 @@
+package view;
+
+import javax.swing.*;
+import java.awt.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.awt.event.KeyEvent;
+import java.awt.event.KeyListener;
+import java.beans.PropertyChangeEvent;
+import java.beans.PropertyChangeListener;
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.ArrayList;
+import java.util.Map;
+
+import interface_adapter.edit_profile.EditProfileController;
+import interface_adapter.edit_profile.EditProfileState;
+import interface_adapter.edit_profile.EditProfileViewModel;
+
+
+public class EditProfileView extends JPanel implements ActionListener, PropertyChangeListener {
+    public final String viewName = "edit profile";
+    private final JTextField changeBioInputField = new JTextField(50);
+    private final JTextField changeSpotifyInputField = new JTextField(15);
+
+    private final EditProfileController editProfileController;
+    private final EditProfileViewModel editProfileViewModel;
+    private final EditBioController editBioController;
+    private final EditBioViewModel editBioViewModel;
+    private final AuthorizeController authorizeController;
+    private final AuthorizeViewModel authorizeViewModel;
+    private final DisplayPlaylistsController displayPlaylistsController;
+    private final DisplayPlaylistsViewModel displayPlaylistsViewModel;
+    private final ChoosePlaylistController choosePlaylistController;
+    private final ChoosePlaylistViewModel choosePlaylistViewModel;
+    private final EditSpotifyHandleController editSpotifyHandleController;
+    private final EditSpotifyHandleViewModel editSpotifyHandleViewModel;
+    private final GoBackController goBackController;
+    private final GoBackViewModel goBackViewModel;
+
+    public EditProfileView(EditProfileController editProfileController, EditProfileViewModel editProfileViewModel,
+                           EditBioController editBioController, EditBioViewModel editBioViewModel,
+                           AuthorizeController authorizeController, AuthorizeViewModel authorizeViewModel,
+                           DisplayPlaylistsController displayPlaylistsController, DisplayPlaylistsViewModel displayPlaylistsViewModel,
+                           ChoosePlaylistController choosePlaylistController, ChoosePlaylistViewModel choosePlaylistViewModel,
+                           EditSpotifyHandleController editSpotifyHandleController, EditSpotifyHandleViewModel editSpotifyHandleViewModel,
+                           GoBackController goBackController, GoBackViewModel goBackViewModel) {
+        this.editProfileController = editProfileController;
```

Comment:
I think this chunk of code is a bit hard to read in terms of style. Is there any way you could group the similar ones together and put spaces between them?

Reply:
Okay! I will work on that.